graylog2-server/src/test/java/org/graylog/plugins/views/search/rest/export/response/ExportTabularResultResponseTest.java

```diff
@@ -81,7 +81,7 @@ void testCreationFromMessageList() throws Exception {
         final ObjectMapper objectMapper = new ObjectMapperProvider().get();
         final byte[] bytes = Resources.toByteArray(Resources.getResource("org/graylog/plugins/views/search/rest/export/response/sample_message_list_result.json"));
         final MessageList.Result messageListResult = objectMapper.readValue(bytes, MessageList.Result.class);
-        final ExportTabularResultResponse response = ExportTabularResultResponse.fromMessageListResult(messageListResult);
+        final ExportTabularResultResponse response = ExportTabularResultResponse.fromMessageListResult(List.of(), messageListResult);
```

Comment:
I guess that you should also add a unit test for a scenario when list of fields is not empty.